keel-sql/src/main/resources/db/changelog/20200928-add-notifier-repo.yml

```
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-notifier-repo
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: notifier
+            columns:
+              - column:
+                  name: scope
+                  type: varchar(255)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: identifier
```

To avoid confusing this with an actual identifier for this table (which this is not), we've used the name `ref` in the `events` table, for instance. I'd recommend doing the same here.
```suggestion
                  name: ref
```

---

fyi for database stuff the suggest changes isn't useful because I need to change all the auto generated jooq stuff. just pointing it out is fine!

---

Yeah, but you need to change this file so that jooq will re-generate the classes... 🙂

---

Ya of course. Pointing it out is very useful. But the "auto commit" stuff that git does is not the complete picture here. I tend to not do the suggestions because I like to just do the change locally to make sure I get everything involved